features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumVitalAppLaunchEventHelper.kt

```diff
@@ -29,6 +29,7 @@ internal class RumVitalAppLaunchEventHelper(
 ) {
     @Suppress("LongMethod")
     fun newVitalAppLaunchEvent(
+        vitalId: String?,
```

Comment:
it is better to make it non-nullable and simply generate each time at the call site: this will avoid nullability and usage of elvis operator.